### PR TITLE
Forbidden resourced handler

### PR DESCRIPTION
**Pull request by JoostDeCupere**

### CODE REVIEW

File: xenit-alfresco/templates/ingress/alfresco-ingress.yaml [hunk 1]
```diff
@@ -63,6 +63,24 @@ spec:
...
       {{- end }}
+      {{- if .Values.forbiddenHandler.enabled }}
+      - path: /alfresco/s/api/solr
+        pathType: Prefix
+        backend:
+          service:
+            name: nginx-403-service
+            port:
+              number: 30403
+      {{- end }}
```
Comment (thijslemmens) [inline xenit-alfresco/templates/ingress/alfresco-ingress.yaml:74]: why end and restart if?

**Review by thijslemmens — CHANGES_REQUESTED** [review]
I'm not convinced that the current naming and description is clear.

File: xenit-alfresco/templates/ingress/alfresco-ingress.yaml (hunk 1 above)
Comment (JoostDeCupere) [reply xenit-alfresco/templates/ingress/alfresco-ingress.yaml]: no reason, removed it

**Review by thijslemmens — APPROVED** [review]
I don't think we need the .enabled but if you disagree you can leave it.

**Review by thijslemmens — CHANGES_REQUESTED** [review]
so also the wcs and wcservice endpoints need to be handled

**Review by thijslemmens — APPROVED** [review]

**Review by hechmi-dammak-xenit — APPROVED** [review]

**Pull request merged**